tee/params.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package tee
+
+import "github.com/ethereum/go-ethereum/common"
+
+type Parameters struct {
+	PowDepth         uint64         // required confirmed block depth
+	PhaseDuration    uint64         // number of blocks of one phase (not epoch)
+	ResponseDuration uint64         // challenge response grace period for operator at end of exit phase
+	InitBlock        uint64         // block at which Erdstall contract was deployed
+	TEE              common.Address // Enclave's public key address
+	Contract         common.Address // Erdstall contract address
+}
+
```

Comment:
For completeness, we should add a constructor that checks the requirements as in the contract.  
Currently only `require(2 * _responseDuration <= _phaseDuration, "responseDuration too long");`.  
But not doing it is also ok, since then the deployment just fails.

Reply:
Yes good point, but I think it's fine if it fails during deployment. The clients would just read those parameters from the deployed contract anyways, where they are guaranteed to be correct.

Reply:
We discussed to let the deployment fail.